zap/src/main/java/org/zaproxy/zap/extension/alert/AlertAddDialog.java

```diff
@@ -205,6 +207,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
                                     alert.setSource(Alert.Source.MANUAL);
                                     // Raise it
                                     extAlert.alertFound(alert, historyRef);
+                                    Stats.incCounter("stats.ui.alert.dialog.add");
```

Comment:
IMO we should do this in the `ExtensionAlert#alertFound` based on the source, the user can also raise alerts manually through the API (though I guess we could have in both places).

Reply:
I was about to change this, but is it worth knowing the source? (dialog vs api)

Reply:
/nudge

Reply:
Are you suggesting using the same stat in the API? Or not add stats there?

Reply:
I'm asking if we want to have two separate stats specifically. Or just one (in the suggested location) that covers both scenarios.

Reply:
I'm good with two.